runtime/lua/json.lua

```diff
@@ -0,0 +1,383 @@
+--
+-- json.lua
```

Comment:
Luv won't spawn threads, but it can run callbacks at times which are not considered vimL-safe, though as long as GC won't be invoked in such a time (potentially, rooting might be incomplete), json_decode shouldn't be a problem.